plugins/org.eclipse.fordiac.ide.product/org.eclipse.fordiac.ide.product.target

```diff
@@ -102,5 +102,39 @@
 				</dependency>
 			</dependencies>
 		</location>
+		<location includeDependencyDepth="infinite" includeDependencyScopes="compile" includeSource="true" missingManifest="generate" type="Maven">
+
+			<feature id="aimirim.fordiac.ide.websocket" label="Websocket Dependencies" provider-name="Aimirim" version="0.1.0.qualifier">
```

Comment:
please name that org.eclipse.fordiac.ide.websocket

Reply:
Renamed in a40b827 as well.